website/docs/concepts/event-bridge/index.md

```diff
@@ -28,6 +26,262 @@ To jump right into building, see the [`EventBus`](./event-bus.md) documentation.
+  id?: string;
+  name: string;
+  age: number;
+  operation: "Create" | "Update";
+  interests: string[];
+}
+
+interface Delete {
+  id: string;
+}
+
+const createOrUpdateOperation = functionless.Function<CreateOrUpdate, void>(
+  createOrUpdateFunction
+);
+const deleteOperation = functionless.Function<Delete, void>(deleteFunction);
+```
+
+The events from before do not match the formats from before, so lets transform them to the structures match.
+
+```ts
+const createOrUpdateEventsTransformed =
+  createOrUpdateEvents.map<CreateOrUpdate>((event) => ({
+    id: event.detail.id,
+    name: event.detail.name,
+    age: event.detail.age,
+    operation: event["detail-type"],
+    interests: event.detail.interests,
+  }));
+
+const deleteEventsTransformed = createOrUpdateEvents.map<Delete>((event) => ({
+  id: event.detail.id,
+}));
+```
+
+## Target other AWS services like Lambda and other Event Buses
+
+Now that we have created rules on our event buses using `when` and transformed those matched events using `map`, we need to send the events somewhere.
+
+We can `pipe` the transformed events to the lambda functions we defined earlier.
+
+```ts
+createOrUpdateEventsTransformed.pipe(createOrUpdateOperation);
+deleteEventsTransformed.pipe(deleteOperation);
+```
 
-There is a default Event Bus in every region of an AWS account. It contains events emitted by your AWS Resources, such as when a Step Function execution completes, or when a scheduled trigger fires.
+What about our young cat lovers? We want to forward those events to our sister team's event bus for processing.
+
+```ts
+const catPeopleBus = functionless.EventBus.fromBus(
+  aws_events.EventBus.fromEventBusArn(this, "catTeamBus", catTeamBusArn)
+);
+
+// Note: EventBridge does not support transforming events which target other event buses. These events are sent as is.
+catPeopleEvents.pipe(catPeopleBus);
+```
+
+## Put Events from other sources
+
+Event Bridge Put Events API is one of the methods for putting new events on an event bus. We support some first party integrations between services and event bus.
+
+Support (See [issues](https://github.com/functionless/functionless/issues?q=is%3Aissue+is%3Aopen+label%3Aevent-bridge) for progress):
+
+- Step Functions
+- App Sync (coming soon)
+- API Gateway (coming soon)
+- More - Please create a new issue in the form `Event Bridge + [Service]`
+
+```ts
+bus = new EventBus(stack, "bus");
+new StepFunction<{ value: string }, void>((input) => {
+  bus({
+    detail: {
+      value: input.value,
+    },
+  });
+});
+```
+
+This will create a step function which sends an event. It is also possible to send multiple events and use other Step Function logic.
+
+> Limit: It is not currently possible to dynamically generate different numbers of events. All events sent must start from objects in the form `{ detail: ..., source: ... }` where all fields are optional.
+
+## Putting it all together.
+
+Lets look at the above all together.
```

Comment:
this doesn't include sending to the bus from SFN, maybe it should be moved up a section